.travis.yml

```diff
@@ -12,23 +12,23 @@
 # See the License for the specific language governing permissions and
```

Comment:
When we get ready to merge can you document the docker lines with a comment as it might not be clear what each line is doing.